scripts/apps/relations/directives/RelatedItemsDirective.ts

```diff
@@ -189,10 +193,13 @@ export function RelatedItemsDirective(authoringWorkspace, relationsService, noti
                 const key = getNextKey(scope.item.associations || {}, scope.field._id);
                 let data = {};
 
-                if ('_type' in item && item._type !== 'archive') {
-                    data[key] = item;
+                if (isInArchive(item)) {
+                    data[key] = {
+                        _id: item._id,
+                        type: item.type, // used to display associated item types
+                    };
                 } else {
-                    data[key] = {_id: item._id};
+                    data[key] = item; // use full item for external items
```

Comment:
What's the use case for "external items". If you want to add an item as related, doesn't it make sense to fetch it and make it internal?

Reply:
that's items from external repos, like scanpix images or ansa photo archive

Reply:
can you add this to the comment in the code?